### PR TITLE
RSDEV-368: Adding MVCIT test as a POC for Fieldmark

### PR DESCRIPTION
**Pull request by rs-nicof**

### CODE REVIEW

**Comment by rs-nicof** [comment]
The model classes will be moved in a later stage of the implementation, when the client module is created

File: src/test/java/com/researchspace/webapp/integrations/fieldmark/FieldmarkMVCIT.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.web.client.RestTemplate;
+
+@WebAppConfiguration
+public class FieldmarkMVCIT extends API_MVC_TestBase {
+
+  private @Autowired UserConnectionManager userConnectionManager;
+  private User user;
```
Comment (rs-fraser) [inline src/test/java/com/researchspace/webapp/integrations/fieldmark/FieldmarkMVCIT.java:30]: It doesn't seem like these 2 fields, or the user init on line 35 are needed.

Comment (rs-nicof) [reply src/test/java/com/researchspace/webapp/integrations/fieldmark/FieldmarkMVCIT.java]: thanks, removed

**Review by rs-fraser — APPROVED** [review]

File: src/test/java/com/researchspace/webapp/integrations/fieldmark/FieldmarkMVCIT.java [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+          + "We leave the test Ignored so we can potentially run it manually by the bearer token")
+  public void testCreateDatasetAndPushFile() throws IOException, URISyntaxException {
+    RestTemplate restTemplate = new RestTemplate();
+    HttpHeaders headers = new HttpHeaders();
+    headers.setContentType(MediaType.APPLICATION_FORM_URLENCODED);
+
+    /* ************
+     ///// SET THE ACCESS TOKEN MANUALLY -  It is in BitWarden /////
+    **************** */
+    String ACCESS_TOKEN = "-----------PASTE_TOKEN_HERE_FROM_BITWARDEN----------";
```
Comment (mKowalski256) [inline src/test/java/com/researchspace/webapp/integrations/fieldmark/FieldmarkMVCIT.java:51]: Bitwarden is our company-specific tool, maybe more generic instructions about obtaining token form Fieldmark would be better for external users who may read it.

**Review by mKowalski256 — APPROVED** [review]
Looks good to me, just a minor comment about token instructions.

**Pull request merged**